src/dvc_objects/fs/generic.py

```diff
@@ -397,3 +409,118 @@ def test_links(
         from_fs.remove(from_file)
 
     return ret
+
+
+def status(
```

Comment:
Should it be `exists`? Trying to understand if it does anything over just being a fast bulk exists method.

Reply:
Will rename it, for the record we also discussed whether or not it belongs in `fs.generic`.

I could move it into `fs.utils` if that's preferred?